assemblies/managing-distributed-workloads.adoc

```diff
@@ -0,0 +1,21 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="managing-distributed-workloads_{context}"]
+= Managing distributed workloads
+
+[role='_abstract']
+
+In {productname-short}, cluster administrators create Kueue resources to configure quota management for distributed workloads.
+Cluster administrators can optionally configure the CodeFlare Operator if they want to change its default behavior.
```

Comment:
I'm not too sure what is meant by this line. Is this in relation to the VAP feature?

Reply:
Ohh okay that's perfect! - I'd say "create Kueue resources" covers this.